utils/include/Utilities.cpp

```diff
@@ -0,0 +1,61 @@
+#include "stdafx.h"
+#include "Utilities.h"
+
+std::string Utilities::GetCurrentDateAndTimeFormatted()
+{
+    //parse to format: dd.mm.yyyy hh:mm:ss
+    std::time_t current_time = std::chrono::system_clock::to_time_t(
+        std::chrono::system_clock::now());
+    std::string date_and_time = ctime(&current_time);
+
+    date_and_time.pop_back();
+    date_and_time = date_and_time.substr(date_and_time.find_first_of(" ") + 1);
+    
+    std::string buff_year = date_and_time.substr(
+        date_and_time.find_last_of(" ") + 1);
+    date_and_time.erase(date_and_time.end() - buff_year.size(), 
+        date_and_time.end());
+
+    std::string buff_month = date_and_time.substr(0, 
+        date_and_time.find_first_of(" "));
+    date_and_time.erase(date_and_time.begin(), date_and_time.begin() + 
+        buff_month.size() + 1);
+
+    if (!HelperDateAndTime::TrySetMonthAsNumber(&buff_month))
+    {
+        return std::string("Couln't parse month to number!");
+    }
+    
+    date_and_time.insert(date_and_time.find_first_of(" "), "." + buff_month);
+    date_and_time.insert(date_and_time.find_first_of(" "), 
+        "." + buff_year + " ");
+
+    return date_and_time;
+}
+
+bool Utilities::HelperDateAndTime::TrySetMonthAsNumber(std::string* p_month)
```

Comment:
if you are passing a pointer to a function you should check it if it not nullptr